spras/dataset.py

```diff
@@ -63,25 +63,47 @@ def load_files_from_dict(self, dataset_dict):
 
         self.label = dataset_dict["label"]
 
-        #Get file paths from config
+        # Get file paths from config
         # TODO support multiple edge files
         interactome_loc = dataset_dict["edge_files"][0]
         node_data_files = dataset_dict["node_files"]
-        #edge_data_files = [""]  # Currently None
+        # edge_data_files = [""]  # Currently None
         data_loc = dataset_dict["data_dir"]
 
-        #Load everything as pandas tables
-        self.interactome = pd.read_table(os.path.join(data_loc,interactome_loc), names = ["Interactor1","Interactor2","Weight"])
+        # Load everything as pandas tables
+        self.interactome = pd.read_table(
+            os.path.join(data_loc, interactome_loc), sep="\t", header=None
+        )
+        num_cols = self.interactome.shape[1]
+        if num_cols == 3:
```

Comment:
Do we document anywhere that if the interaction file has three columns we assume it is undirected? We need to update `input/README.md` to describe these changes to the input edge file format and show an example of four column format, keeping the three column example.

Reply:
We're checking to see if there are three or four columns, but I'm not sure where we are throwing an error if a fourth column in the network file has missing data or random values.
Right now, it appears like different areas of the code or algorithm code are raising the issues, but there isn't a specific location where the error is being produced. 

Should I include anything that checks for this and throws an error, or should I leave it up to the user to ensure their file is correct once I update the documentation?

Reply:
That's a good point. We could check the fourth column to ensure it does not have missing values and that it only has "U" or "D" as values.